src/components/Reports/GDPRPermissions/SubscriberGdprPermissions.js

```diff
@@ -95,6 +108,17 @@ const SubscriberGdprPermissions = ({
           </table>
         )}
       </div>
+
+      {downloadPermissionHistoryUrl && (
+        <DownloadLink className="dp-cta-links">
+          <a href={downloadPermissionHistoryUrl}>
+            <i className="ms-icon icon-download"> </i>
+            <span className="m-l-6 align-middle">
+              <FormattedMessage id="subscriber_gdpr.download_permission_history" />
+            </span>
+          </a>
+        </DownloadLink>
+      )}
```

Comment:
If `downloadPermissionHistoryUrl` is false, does it render `"false"`? 

Maybe using the ternary operator is a better option, see https://kentcdodds.com/blog/use-ternaries-rather-than-and-and-in-jsx

Reply:
You're right. Excellent article, thanks for sharing.